enclave/sgx/revocationinfo.c

```diff
@@ -81,20 +82,30 @@ oe_result_t oe_get_revocation_info(oe_get_revocation_info_args_t* args)
         /* tcb_info */
         if (in.tcb_info_size < out.tcb_info_size)
         {
-            if (!(in.tcb_info = realloc(in.tcb_info, out.tcb_info_size)))
+            if (!(temp = realloc(in.tcb_info, out.tcb_info_size)))
+            {
                 OE_RAISE(OE_OUT_OF_MEMORY);
+            }
+            else
+            {
+                in.tcb_info = temp;
+            }
```

Comment:
What is the reason for doing this? There is no memory safety issue in the original implementation that i can see.